src/client/lobby/lobby.js

```diff
@@ -19,21 +17,6 @@ export class Lobby extends Component {
         configure(this.props.refreshLobby, this.props.refreshRooms, this.props.refreshRoomUsers);
```

Comment:
This configure function could go into the CloakService as well :clock1: (no cloaks in markdown :crying_cat_face: )